src/pybind/mgr/rook/module.py

```diff
@@ -434,6 +436,14 @@ def remove_service(self, service_name: str) -> str:
         else:
             raise orchestrator.OrchestratorError(f'Service type {service_type} not supported')
 
+    def zap_device(self, host: str, path: str) -> OrchResult[str]:
+        try:
+            self.rook_cluster.create_zap_job(host, path)
+        except Exception as e:
+            logging.error(e)
+            return OrchResult(None, "Unable to zap device" + str(e.with_traceback(None)))
```

Comment:
```suggestion
            return OrchResult(None, "Unable to zap device " + str(e.with_traceback(None)))
```

Or is `OrchResult` printing nicely already?